Bukkit/src/main/java/com/plotsquared/bukkit/BukkitMain.java

```diff
@@ -904,7 +906,11 @@ public final ChunkGenerator getDefaultWorldGenerator(@NotNull final String world
         getServer().getPluginManager().registerEvents(new ProjectileEventListener(), this);
         getServer().getPluginManager().registerEvents(new EntitySpawnListener(), this);
         if (PaperLib.isPaper() && Settings.Paper_Components.PAPER_LISTENERS) {
-            getServer().getPluginManager().registerEvents(new PaperListener(), this);
+            if (Reflection.getVersion().startsWith("v1_13")) {
```

Comment:
Just a random thought but wouldn't it be better to use Class.forName and lookup the missing TE class here? Doesn't really matter tho